website/docs/components/segmented-group/partials/code/component-api.md

```diff
@@ -1,29 +1,79 @@
 ## Component API
 
+### SegmentedGroup
+
 <Doc::ComponentApi as |C|>
+  <C.Property @name="<[S].Button>" @type="yielded component">
```

Comment:
`[SG]`?

Reply:
Same as above, we've documented in the "How to use" as `<Hds::SegmentedGroup as |S|>`. Should I change it also there and use `SG` for both?